packages/security/src/SecurityService.ts

```diff
@@ -0,0 +1,7 @@
+import { SecurityCheck, SecurityCheckResult } from "./deviceTrust";
+
+export class SecurityService {
+  public check(check: SecurityCheck): Promise<SecurityCheckResult> {
+    return check.test();
+  }
```

Comment:
I think we should also support run an list of checks

Reply:
@wei-lee There is another ticket to address the list of checks properly. Mainly to make this PR as quick as possible to unblock all other tasks in the story.